src/applications/mhv/medical-records/containers/Allergies.jsx

```diff
@@ -33,7 +37,76 @@ const Allergies = () => {
     [dispatch],
   );
 
-  const download = () => {};
+  const generateAllergiesPdf = async res => {
+    const pdfData = {
+      headerLeft: 'Roberts, Jesse',
+      headerRight: 'Date of birth: January 1, 1970',
+      footerLeft: `Report generated by My HealtheVet and VA on ${moment().format(
+        'LL',
+      )}`,
+      footerRight: 'Page %PAGE_NUMBER% of %TOTAL_PAGES%',
+      title: 'Allergies',
+      subject: 'VA Medical Record',
+      preface:
+        'Your allergies list may not be complete. If you have any questions about your information, visit the FAQs or contact your VA Health care team.',
+      results: {
+        items: [],
+      },
+    };
+
+    res.forEach(item => {
+      pdfData.results.items.push({
+        header: item.name,
+        items: [
+          {
+            title: 'Date entered',
+            value: item.date || emptyField,
+            inline: true,
+          },
+          {
+            title: 'Reaction',
+            value: processList(item.reaction),
+            inline: true,
+          },
+          {
+            title: 'Type of allergy',
+            value: item.type || emptyField,
+            inline: true,
+          },
+          {
+            title: 'VA drug class',
+            value: item.drugClass || emptyField,
+            inline: true,
+          },
+          {
+            title: 'Location',
+            value: item.location || emptyField,
+            inline: true,
+          },
+          {
+            title: 'Observed or reported',
+            value: item.observed ? 'Observed' : 'Reported',
+            inline: true,
+          },
+          {
+            title: 'Provider notes',
+            value: processList(item.notes),
+            inline: !item.notes,
+          },
+        ],
+      });
+    });
+
+    try {
+      await generatePdf('medicalRecords', 'allergies_report', pdfData);
+    } catch (error) {
+      // Error logging/presentation goes here...
```

Comment:
This should have sentry logging 🙂 thanks!

Reply:
I have not used sentry before. Do you know of any good examples I could look at?

Reply:
I added sentry logging modeled after Robert's implementation